src/components/Word.js

```diff
@@ -39,4 +39,4 @@ export default class Word extends Component{
     render(){
     return (<div><p>Generated word: {this.state.word}</p><p>Udpate result: {this.state.update_result}</p></div>);
     }
-}
\ No newline at end of file
+}
```

Comment:
purpose of adding a new line?

Reply:
Sometimes the compiler may fail due to the lack of a newline at the end of the file, especially on linux.